### ISSUE TITLE
Set "Auth: Inherited" as default on openapi import

### ISSUE THREAD

**Issue by romst** [usebruno/bruno#2315]
### I have checked the following:

- [X] I've searched existing issues and found nothing related to my issue.

### Describe the feature you want to add

I love the option to set the default authentication on collection level. However, when I import an openapi spec, it doesn't set "inherit" as default. For now, I import the openapi, export it again, modify the json, then import it with "inherit" on all requests.

A better default on openapi import for auth would be to just use "inherit" as auth strategy.

### Mockups or Images of the feature

![image](https://github.com/usebruno/bruno/assets/10211736/35932159-d3ac-4c76-aaf6-a31795d4de1f)

**Comment by Wapax** [comment]
I really want it too !

**Comment by edbrannin** [comment]
This issue says "on openapi import", but I think Inherit would be a good default for _all_ new requests.

**Comment by cmellazchy** [comment]
I agree that `Inherit` should be the default auth mode for all new requests in collections. +1

**Comment by draial** [comment]
yes please :)

**Comment by helloanoop** [comment]
We've allocated time for this to be worked upon and delivered in March.
We'll be reviewing the community PR - https://github.com/usebruno/bruno/pull/3436

**Comment by mukjo96** [comment]
#4730
